rules.md

```diff
@@ -511,140 +422,177 @@ Ultimately, decisions on whether some output meets the requirements specified in
 
-Therefore we can verify that the total funds spent listed on the [Dash Blockchain](https://insight.dash.org/insight/address/XbFb9b1qaoLykngDbUwBVBFwSHuwQRhSqc) tally to total task claims awarded in the [Awarded Claims List](https://docs.google.com/spreadsheets/d/1mhXlo4ylqWLLSYN4MGiLWlp7Gq3jrsDt0kB701dwMNU/edit#gid=0) to ensure that funds aren't being spent from the wallet arbitrarily by validating that:
+Each Strategist's published address(es) carries funds in a rolling budget, similar to the general budget.  Strategists have full autonomy over their budget spending, so long as they serve the Incubator's vision and mission, and are accounted for according to the [auditing rules](#53-auditing).
 
- - Total funds spent via the Blockchain is equal to the total Dash sent minus the total Dash received from the [Incubator Wallet](#63-accounting).
- - Total funds spent on awarded Task Claims is equal to the total Dash spent in columns H (Reward) and N (Admin reward) in the [Awarded Claims List](https://docs.google.com/spreadsheets/d/1mhXlo4ylqWLLSYN4MGiLWlp7Gq3jrsDt0kB701dwMNU/edit#gid=0).
+When a Strategist leaves or is removed from the Incubator their budget is returned to the General Budget.
 
-Note that each Task lists the transaction id, which can also be parsed individually to verify the total amount.
+## 5.3 Auditing
 
-### 5.4.2 Auditing Work
+Anyone can perform a full public audit (#14-full-transparency) of the Incubator at any time to check that all work is being conducted as per these Rules, and verify the rewards issued to individual Incubator users for that work via the Dash blockchain.
+
+### 5.3.1 Auditing Accounts
+
+All spending by the Incubator is tracked via [Task Claims](#26-claims) approved by [Admins](#4-admins), with associated transactions accounted for in the [Incubator Accounting Spreadsheet](https://docs.google.com/spreadsheets/d/1mhXlo4ylqWLLSYN4MGiLWlp7Gq3jrsDt0kB701dwMNU/edit?usp=sharing).
+
+- The Balance tab shows all incoming funds and compares it with all approved Task Claims and the current blockchain balance. 
+- The Claims tab lists transaction ids for all rewarded claims which can be parsed to verify the reward amounts.
+
+### 5.3.2 Auditing Work
 
 All work must be conducted inline with the Rules in this document and in the public domain.  Therefore anyone can audit how well the work being conducted in the Incubator is adhering to these rules at any time.
 
 As a guideline, the following aspects of work are key areas to ensure the Rules are being implemented effectively.
 
  - Admins are pricing work fairly
  - Admins are interpreting the rules effectively
- - Concepts being approved are inline with the [Strategy](#13-strategy)
+ - Concepts being approved are inline with our [Mission](#11-mission) and [Vision](#12-dash-vision)
```

Comment:
Concept approval is inline with.... 
OR
Concepts that have been approved are ...

(active vs passive voice - easier to read)

Reply:
Fixed in https://github.com/dashincubator/dash-incubator-rules/pull/11/commits/c4c94a5e12f23b7efe0e79e6b4c694488486ce9b